Makefile

```diff
@@ -121,9 +121,9 @@ test-integration: manifests fmt vet envtest ginkgo ## Run integration tests.
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
 .PHONY: test-e2e
-test-e2e: manifests generate fmt vet envtest ginkgo kind-image-build
-	E2E_KIND_VERSION=$(E2E_KIND_VERSION) KIND_CLUSTER_NAME=$(KIND_CLUSTER_NAME) KIND=$(KIND) KUBECTL=$(KUBECTL) KUSTOMIZE=$(KUSTOMIZE) GINKGO=$(GINKGO) USE_EXISTING_CLUSTER=$(USE_EXISTING_CLUSTER) IMAGE_TAG=$(IMG) ./test/e2e/e2e-test.sh
-	
+test-e2e: kustomize manifests generate fmt vet envtest ginkgo kind-image-build
```

Comment:
qq, do we need "generate" step here?

Reply:
let's keep the same with integration-test, removed in https://github.com/kubernetes-sigs/lws/pull/81